apps/siwe-delegate-manager/src/App.tsx

```diff
@@ -0,0 +1,96 @@
+import Typography from '@material-ui/core/Typography'
+import Paper from '@material-ui/core/Paper'
+import styled from 'styled-components'
+
+import { useDelegateRegistry } from 'src/store/delegateRegistryContext'
+import Header from 'src/components/header/Header'
+import AddressLabel from 'src/components/address-label/AddressLabel'
+import DelegateForm from 'src/components/delegate-form/DelegateForm'
+import DelegationTable from 'src/components/delegation-table/DelegationTable'
+import DelegationHistoryTable from 'src/components/delegation-history-table/DelegationHistoryTable'
+
+function App() {
+  const { delegateRegistryContract, setDelegate } = useDelegateRegistry()
+
+  const handleAddNewDelegate = async (space: string, delegateAddress: string) => {
+    await setDelegate(space, delegateAddress)
+  }
+
+  return (
+    <>
+      {/* App Header */}
+      <Header />
+
+      {/* Delegate Registry Title & Form */}
+      <AppWrapper>
+        <TitleContainer>
+          <Typography component="h2" variant="h5" gutterBottom>
+            Sign-In With Ethereum Delegate Registry Contract
+          </Typography>
+
+          <Typography component="h3" variant="h6" gutterBottom>
+            <AddressLabel
+              address={delegateRegistryContract?.address || ''}
+              showCopyIntoClipboardButton
+              showBlockExplorerLink
+              ariaLabel="Sign-In With Ethereum Delegate Registry contract"
+            />
+          </Typography>
+
+          <DescriptionContainer>
+            <Typography gutterBottom>
+              The Sign-In With Ethereum Delegate Manager App allows owners of a Safe to delegate
+              sign-in authority for applications that have integrated with Sign-In With Ethereum for
+              Safes <a href="https://login.xyz/">Sign-In With Ethereum for Safes</a> to one or
```

Comment:
I think that we can add a `target="_blank"` to this link to open the Sign-In With Ethereum page on a new tab?